admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/controller/DynamicFieldControllerTest.java

```diff
@@ -131,18 +131,48 @@ public void t001createDynamicFieldTest() throws Exception {
 								.andReturn(),
 						null);
 	}
+
+	@Test
+	@WithUserDetails("global-admin")
+	public void createDynamicFieldFailTest() throws Exception {
+		JsonNode fieldVal = mapper.readTree("{\"code\":\"%^%$\",\"value\":\"ooo\"}");
+		dynamicFieldDtoReq.getRequest().setFieldVal(fieldVal);
+		dynamicFieldDtoReq.getRequest().setName("bloodtype");
+		MasterDataTest
+				.checkResponse(
+						mockMvc.perform(
+										MockMvcRequestBuilders.post("/dynamicfields").contentType(MediaType.APPLICATION_JSON)
+												.content(mapper.writeValueAsString(dynamicFieldDtoReq)))
+								.andReturn(),
+						"KER-MSD-999");
+	}
 	
+	@Test
+	@WithUserDetails("global-admin")
+	public void createDynamicFieldFailTest1() throws Exception {
+		JsonNode fieldVal = mapper.readTree("{\"code\":\"avj\",\"value\":\"%^%$\"}");
+		dynamicFieldDtoReq.getRequest().setFieldVal(fieldVal);
+		dynamicFieldDtoReq.getRequest().setName("bloodtype");
+		MasterDataTest
+				.checkResponse(
+						mockMvc.perform(
+										MockMvcRequestBuilders.post("/dynamicfields").contentType(MediaType.APPLICATION_JSON)
+												.content(mapper.writeValueAsString(dynamicFieldDtoReq)))
+								.andReturn(),
+						"KER-MSD-999");
+	}
+
 	@Test
 	@WithUserDetails("global-admin")
 	public void t001createDynamicFieldTest1() throws Exception {
```

Comment:
Let us follow the below convention when naming the test case
<methodNameBeingTested>_<testcase>_<testCaseStatus>
Eg: createDynamicField_withAllValidInput_thenPass
It clearly explains what is tested and its possible output.

Reply:
changed the testcase names